.github/workflows/sql-workbench-test-and-build-workflow.yml

```diff
@@ -1,11 +1,39 @@
 name: SQL Workbench Test and Build
 
-on: [pull_request, push]
+on:
+  push:
+    branches-ignore:
+      - 'dependabot/**'
+    paths:
+      - 'build/**'
+      - 'build-tools/**'
+      - 'build.gradle/**'
+      - 'buildSrc/**'
+      - 'common/**'
+      - 'config/**'
+      - 'core/**'
+      - 'doctest/**'
+      - 'gradle/**'
+      - 'gradle.properties/**'
+      - 'gradlew/**'
+      - 'gradlew.bat/**'
+      - 'integ-test/**'
+      - 'legacy/**'
+      - 'lombok.config/**'
+      - 'opensearch/**'
+      - 'plugin/**'
+      - 'ppl/**'
+      - 'protocol/**'
+      - 'scripts/**'
+      - 'settings.gradle/**'
+      - 'sql/**'
+      - 'workbench/**'
+      - '.github/workflows/sql-workbench-test-and-build-workflow.yml'
 
 env: 
   PLUGIN_NAME: query-workbench-dashboards
   OPENSEARCH_VERSION: 'main'
-  OPENSEARCH_PLUGIN_VERSION: 2.4.0.0
+  OPENSEARCH_PLUGIN_VERSION: 2.3.0.0
```

Comment:
Do you want to revert Peng's change to 2.4?

Reply:
Thank you, fixed in 1d935e6c.